src/commands/item.js

```diff
@@ -438,11 +439,32 @@ class Export extends Command {
 
       if (!path) return
 
-      const data =
-        yield call(db.seq, conn => mod.item.export(conn, [payload.id]))
+      const [resources, props] = yield select(state => {
+        const itms = pick(state.items, ids)
+        const templateIDs = Object.values(itms).map(itm => itm.template)
+        const templates = pick(state.ontology.template, templateIDs)
+        let results = []
+        for (const t in templates) {
+          const template = templates[t]
+          results.push({
+            template,
+            items: Object.values(itms).filter(i => i.template === t),
+            metadata: state.metadata,
+            photos: state.photos
+          })
+        }
+        return [results, state.ontology.props]
+      })
+
+      const results = yield call(groupedByTemplate, resources, props)
 
-      yield call((...args) => write(...args), path, data, { flags: 'w' })
+      const data = JSON.stringify(results, null, 2)
 
+      if (path === ':clipboard:') {
+        yield call(clipboard.writeText, data)
+      } else {
+        yield call((...args) => write(...args), path, data, { flags: 'w' })
```

Comment:
This is odd: why not just call `write` here? I realize you inherited this from me, but it's weird that extra function is needed -- must have to do with the way `write` is promisified or by how `call` passes the arguments on. No big deal now, but bonus points if you look into what's going on there! : )

Reply:
The write function is promisified, this is likely why the standard call method did not work here. the redux-saga docs mention a `cps` method to be used in this very case (https://redux-saga.js.org/docs/basics/DeclarativeEffects.html) so I've switched to that, it uses old-school callbacks underneath.